guides/source/routing.md

```diff
@@ -517,7 +532,7 @@ You can also use [`url_for`][ActionView::RoutingUrlFor#url_for] with a set of ob
 <%= link_to 'Ad details', url_for([@magazine, @ad]) %>
 ```
 
-In this case, Rails will see that `@magazine` is a `Magazine` and `@ad` is an `Ad` and will therefore use the `magazine_ad_path` helper. In helpers like `link_to`, you can specify just the object in place of the full `url_for` call:
+In this case, Rails will see that `@magazine` is a `Magazine` and `@ad` is an `Ad` and will therefore use the `magazine_ad_path` helper. An even shorter way to write that `link_to` is to specify just the object instead of the full `url_for` call:
 
```

Comment:
Maybe it would be useful to show the link that these examples generate?

Reply:
added